packages/react-dialog/src/components/Dialog/useDialogStyles.ts

```diff
@@ -1,28 +1,49 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import type { DialogState } from './Dialog.types';
+import { tokens } from '@fluentui/react-theme';
 
 export const dialogClassName = 'fui-Dialog';
 
-/**
- * Styles for the root slot
- */
 const useStyles = makeStyles({
   root: {
-    // TODO Add default styles for the root element
+    backgroundColor: tokens.colorNeutralBackground1,
+    color: tokens.colorNeutralForeground1,
+    width: '100%',
+    maxWidth: '600px',
+    height: 'fit-content',
+    maxHeight: '100vh',
+    boxShadow: tokens.shadow64,
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    ...shorthands.borderRadius('8px'),
+    ...shorthands.margin('auto'),
+    ...shorthands.border('1px', 'solid', tokens.colorTransparentStroke),
+    ...shorthands.overflow('auto'),
+    zIndex: 1300, //TODO: hardcoded z-index value might need to change
```

Comment:
should this be a token? @miroslavstastny

Reply:
can they do it globally for all instances of Dialog? Or will each developer need to add their own style overrides?